discovery-handler-modules/coap-discovery-handler/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "coap-discovery-handler"
+version = "0.6.8"
+authors = ["Jiayi Hu <jiayi.ghu@gmail.com>"]
+edition = "2018"
+
+[dependencies]
+akri-discovery-utils = { path = "../../discovery-utils" }
+anyhow = "1.0.41"
+async-trait = "0.1.51"
+coap = { git = "https://github.com/Covertness/coap-rs" }
+coap-lite = "0.5"
+env_logger = "0.8.4"
+log = "0.4.14"
+serde = "1.0.104"
+serde_json = "1.0.45"
+serde_derive = "1.0.104"
+serde_yaml = "0.8.17"
+tokio = { version = "0.2", features = ["rt-threaded", "sync", "time", "stream", "fs", "macros", "uds"] }
```

Comment:
Nit: We recently went to tokio 1.0. Could you bump to that version? It isnt required since it is not embedded in the agent.